test/utils/utils.cpp

```diff
@@ -0,0 +1,46 @@
+// evmone: Fast Ethereum Virtual Machine implementation
+// Copyright 2023 The evmone Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "utils.hpp"
+
+namespace evmone::test
+{
+
+/// Translates tests fork name to EVM revision
```

Comment:
This doc comment should be placed in the function declaration in the header file.